dealers_choice.js

```diff
@@ -11,26 +11,313 @@ app.use(staticMiddleWare);
 
 app.get("/", (request, response) => {
   const entries = postBank.list();
-  const html = `
+  const filteredDificultyPeaks = postBank.getAllPeaksFromDifficulty;
+  let randomIdx = () => Math.floor(Math.random() * entries.length);
+  const homeHtml = `
   <!DOCTYPE html>
   <html lang="en">
     <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>The Fourteeners</title>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>The Fourteeners</title>
+      <link rel="stylesheet" href="/style.css" />
+      <script src="https://kit.fontawesome.com/dc998fff98.js" crossorigin="anonymous"></script>
+      <link rel="preconnect" href="https://fonts.googleapis.com">
+      <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+      <link href="https://fonts.googleapis.com/css2?family=Permanent+Marker&display=swap" rel="stylesheet">
     </head>
     <body>
-      <div class="nav">
-      <ul>
-        <li><a href="#home">Home</a></li>
-        <li><a href="#news">News</a></li>
-        <li><a href="#contact">Contact</a></li>
-        <li style="float:right"><a class="active" href="#about">About</a></li>
-      </ul>
+      <div id="nav">
+          <ul id="sourceInfo">
+            <li>Project by Stephen Wong at Full Stack Academy</li>
+            <li style="float:right"><span>Data Source: <a href="https://www.14ers.com/" target="_blank">14ers.com</a></span></li>
+          </ul>
+          <ul>
+            <li class="link" style="float:right"><a class="active" href="https://github.com/stephen-wong-tx/dealers_choice_js" target="_blank">About</a></li>
+            <li id="icon"><a href="/"><img src="/fourteeners-home-alt.png" alt="The number 14 in front of a mountain range drawing"></a></li>
+          </ul>
       </div>
+      <div id="main-content">
       <h1>The Fourtneeners</h1>
+      <p class="description"> <strong>Fourteener</strong> fȯr-ˈtēn-ər<br /> A mountain peek with an elevation of at least 14,000 ft (4267 m).</p>
```

Comment:
pls refrain from using <strong> or <br />- you can use css/html for that!
for <strong> you can use
```html
<span id="bold">Fourteener</span>
```
```css
#bold: {
  font-weight: 500;
}
```

for <br /> you can use 
```css
margin: some number;
```